### PR TITLE
Make math.js more translation friendly

### PR DESCRIPTION
**Pull request by LogGame-HU-DEaN**

### CODE REVIEW

File: src/locales/en-US/errors.json [hunk 1]
```diff
@@ -10,5 +10,9 @@
...
   "notPlaying": "I ain't playing anything!",
   "missingParameters": "You need to give me parameters!",
-  "nsfwOnly": "This can only be done in channels marked as NSFW."
-}
\ No newline at end of file
+  "nsfwOnly": "This can only be done in channels marked as NSFW.",
+  "mathEvaluation": "An error occurred while evaluating the math expression",
+  "mathExpression": "Invalid math expression",
+  "mathFailedCalculation": "Failed math calculation",
+  "error": "Error"
```
Comment (Doges) [inline src/locales/en-US/errors.json]: Put that "error" in 'commons.json'

Comment (pedrofracassi) [reply src/locales/en-US/errors.json]: Can't we use `errors:generic` instead?

File: src/commands/utility/math.js [hunk 2]
```diff
@@ -7,29 +7,29 @@ module.exports = class Math extends Command {
...
         embed.setColor(Constants.ERROR_COLOR)
-          .setTitle('An error occurred while evaluating the math expression')
+          .setTitle(t('errors:mathEvaluation'))
           .setDescription(error.stack)
       } finally {
         if (isNaN(parseFloat(result))) {
           embed.setColor(Constants.ERROR_COLOR)
-            .setTitle('Invalid math expression')
-            .setDescription(`**Usage:** ${process.env.PREFIX}${this.name} <expression>`)
+            .setTitle(t('errors:mathExpression'))
```
Comment (pedrofracassi) [inline src/commands/utility/math.js]: This should be inside `commands:math`

File: src/commands/utility/math.js [hunk 3]
```diff
@@ -7,29 +7,29 @@ module.exports = class Math extends Command {
...
+        .setDescription(`**${t('commons:usage')}:** ${process.env.PREFIX}${this.name} ${t('commands:math.commandUsage')}`)
     } else {
       try {
         result = math.eval(args.join(' '))
       } catch (error) {
-        this.client.log(`Failed math calculation ${args.join(' ')}\nError: ${error.stack}`, this.name)
+        this.client.log(`${t('errors:mathFailedCalculation')} ${args.join(' ')}\n${t('commons:error')}: ${error.stack}`, this.name)
         embed.setColor(Constants.ERROR_COLOR)
-          .setTitle('An error occurred while evaluating the math expression')
+          .setTitle(t('errors:mathEvaluation'))
```
Comment (pedrofracassi) [inline src/commands/utility/math.js]: This should be inside `commands:math`

**Comment by Doges** [comment]
solve the conflict @LogGame-HU-DEaN

**Comment by bolsomito** [comment]
Being fixed on #95

**Pull request closed without merging**